platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/components/EventIntake.java

```diff
@@ -190,6 +190,8 @@ public void addEvent(final EventImpl event) {
             phaseTimer.activatePhase(EventIntakePhase.ADDING_TO_HASHGRAPH);
             final List<ConsensusRound> consRounds = consensus().addEvent(event);
 
+            shadowGraph.setMinimumGenerationNonAncient(consensus().getMinGenerationNonAncient());
```

Comment:
does the same thing need to happen with the new intake?

Reply:
Ah, yes. You are correct. Having two pathways makes things so much more confusing. We would have noticed this pretty quickly in a longer test, symptom would have been an OOM. Apparently wasn't a fast enough leak to sink our 5 minute abbreviated panel tests.